src/ui/widgets/searchedit.cpp

```diff
@@ -154,7 +160,10 @@ QString SearchEdit::currentCompletion(const QString &text) const
 
 int SearchEdit::queryStart() const
 {
-    const Zeal::SearchQuery currentQuery = Zeal::SearchQuery::fromString(text());
+    const Zeal::SearchQuery currentQuery = m_registry != nullptr
```

Comment:
Is it possible not to depend on `DocsetRegistry` here? If not, we can make `m_registry` required, and then just complain and return 0 if it's not set.